### PR TITLE
backup: test backup and import

### PR DESCRIPTION
**Pull request by overvenus**
###  What have you changed?

This PR adds a test about backup a region into sst files and restore it with importer.

###  What is the type of the changes?

- Improvement (a change which is an improvement to an existing feature)

###  How is the PR tested?

Unit test.

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No.

###  Does this PR affect `tidb-ansible`?

No.

### CODE REVIEW

**Comment by 5kbpers** [comment]
CI failed

**Comment by overvenus** [comment]
PTAL, thanks!

File: components/backup/tests/integrations/test_backup.rs [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+            f.finish().unwrap();
+        }
+
+        // Make ingest command.
+        let mut ingest = Request::default();
+        ingest.set_cmd_type(CmdType::IngestSst);
+        ingest.mut_ingest_sst().set_sst(m.clone());
+        let mut header = RaftRequestHeader::default();
+        let leader = suite.context.get_peer().clone();
+        header.set_peer(leader.clone());
```
Comment (NingLin-P) [inline components/backup/tests/integrations/test_backup.rs]: This `clone()` maybe can avoid.

File: components/backup/tests/integrations/test_backup.rs [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+        end_key: Vec<u8>,
+        backup_ts: u64,
+        path: String,
+    ) -> future_mpsc::UnboundedReceiver<BackupResponse> {
+        let mut req = BackupRequest::new();
+        req.set_start_key(start_key);
+        req.set_end_key(end_key);
+        req.start_version = backup_ts;
+        req.end_version = backup_ts;
+        req.set_path(path.clone());
```
Comment (NingLin-P) [inline components/backup/tests/integrations/test_backup.rs]: This clone() maybe can avoid.

**Pull request merged**